apps/client/src/pages/StepperPage.jsx

```diff
@@ -0,0 +1,248 @@
+// THIS PAGE IS FOR DEMO PURPOSES ONLY
+// DONT REVIEW THIS PAGE EXTENSIVELY
+
+import { Heading, Paragraph } from "@datapunt/asc-ui";
+import React, { useState } from "react";
+import { Helmet } from "react-helmet";
+
+import Layout from "../checker/components/Layouts/DefaultLayout";
+import Nav from "../checker/components/Nav";
+import {
+  StepByStepItem,
+  StepByStepNavigation,
+} from "../checker/components/StepByStepNavigation";
+import DebugDecisionTable from "../components/DebugDecisionTable";
+import withTopic from "../hoc/withTopic";
+
+/**
+ *
+ * To fix:
+ * - add aria-roles
+ * - write tests
+ *
+ * Questions to designer:
+ * - circles desktop 36px?
+ * - highlightActive on mobile?
+ *
+ *
+ * See: *
+ * - https://formulier.amsterdam.nl/thema/bestuur-organisatie/stadsdelen/centrum/aanmelden-spreekuur/Uw-gegevens/ *
+ * - https://designsystem.amsterdam.nl/7awj1hc9f/p/0659fc-step-by-step-navigation
+ *
+ *
+ */
+
+const StepperNav = ({
+  step,
+  stepper,
+  updateStepper,
+  showPrev = true,
+  showNext = true,
+}) =>
+  step === stepper ? (
+    <>
+      <Nav
+        onGoToPrev={() => updateStepper(stepper - 1)}
+        onGoToNext={() => updateStepper(stepper + 1)}
+        {...{ showPrev, showNext }}
+      />
+    </>
+  ) : null;
+
+const StepperPage = ({ checker, topic }) => {
+  const [stepper1, updateStepper1] = useState(1);
```

Comment:
maybe stepper1 is not the best name? 
But do we still need this state after merge? Or do we listen the questionIndex from the context?